src/libertem/api.py

```diff
@@ -95,8 +96,12 @@ def load(self, filetype: str, *args, **kwargs) -> DataSet:
 
     load.__doc__ = load.__doc__ % {"types": ", ".join(filetypes.keys())}
 
-    def create_mask_job(self, factories, dataset, use_sparse=None,
-                        mask_count=None, mask_dtype=None, dtype=None) -> ApplyMasksJob:
+    MaskArrayType = Union[np.ndarray, sp.coo.coo_matrix, sp.dok.dok_matrix]
```

Comment:
This type alias should either be at the topmost level of the file (i.e. not as an attribute of the `Context` class, but as an attribute of the module itself), or even better, it could be moved to a more specific module, for example `libertem.masks` or `libertem.udf.masks`, from where it can be used in the future to type all the other functions that take this form of mask array.

(As a description why it should be moved: this exact type is used in quite a few places, like `libertem.udf.masks`. Now one could think: just let `MaskArrayType` live in `libertem.api` and import that from `libertem.udf.masks`, but this would create an import cycle, in this case via the analysis module. But if `MaskArrayType` lives in `libertem.masks`, you would only have imports going in one direction. No more circular imports!)